app/helpers/application_helper.rb

```diff
@@ -11,7 +11,7 @@ def flash_class(level)
   def title
     translate(
       :"titles.#{controller_name}.#{action_name}",
-      id: resource && resource.id,
+      id: defined?(resource) && resource && resource.id,
```

Comment:
maybe we should pass the resource, controller_name and action_name as as parameters to this method. Then we don't need to check for `defined?(resource)`